src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return { count: state.count + 1}
```

Comment:
It could be GitHub's styling but in addition to trying out the prefix operator `++`, you should indent the return statements inside you switch.
```suggestion
    return { count: ++state.count }
```